### PR TITLE
[clang-3.4-debuginfo-statistics] Fix the LNT submissions run order

### PR DESCRIPTION
**Pull request by azharudd**
The run order should be the GIT_DISTANCE of the stage 1 compiler which is being used to build the historic compiler.

### CODE REVIEW

File: zorg/jenkins/jobs/jobs/clang-3.4-debuginfo-statistics
Comment (justice-adams-apple) [inline zorg/jenkins/jobs/jobs/clang-3.4-debuginfo-statistics:92]: I this still downloading the toolchain package?

**Review by justice-adams-apple — APPROVED** [review]
LGTM pending small question

**Comment by Michael137** [comment]
Thanks for addressing this! Makes sense to me. I found this bit of the pipeline suspicious when I was looking into doing the same for the lldb-statistics bot.

I'll defer to @justice-adams-apple for the review

File: zorg/jenkins/jobs/jobs/clang-3.4-debuginfo-statistics
Comment (azharudd) [reply zorg/jenkins/jobs/jobs/clang-3.4-debuginfo-statistics:92]: Yes, it should. It points the `ARTIFACT` to the one listed in `last_good_build.properties` and that is fetched in the next line.

**Pull request merged**
